### ISSUE TITLE
NeonServices/NeonServiceMap shouldn't be in Neon.Common

### ISSUE THREAD

**Issue by jefflill** [nforgeio/neonKUBE#1136]
I just noticed that the `NeonServices` and `NeonServiceMap` classes are defined in **Neon.Common**.  This doesn't make a lot of sense for two reasons:

1. **Neon.Common** is intended to be used pretty much everywhere in our code and we want to keep it as small as possible so we don't have extra stuff that's not needed for a phone app, etc.  Ultimately, we also hope that other developers start using this, so we don't want extra junk in their apps either.

2. This also doesn't make sense for just for versioning reasons.  The neonLIBRARY and neonKUBE/DESKTOP release life-cycle are different already.  We don't want to be forced to do a neonLIBRARY release just because we want to add another service to the neon service map.  This belongs in one of the **Neon.Kube.*** libraries, probably **Neon.Kube** for now.  I could also see adding a **Neon.Kube.Common** library where this could live such that our services could reference this without having to bring in the full **Neon,Kube** assembly with lots of setup stuff that's probably not necessary.

One idea is to refactor **Neon.Kube** into two libraries like:

* **Neon.Kube:** Holds common neonKUBE related code like the the service map and perhaps things like `KubeHelper`
* **Neon.Kube.Setup:** Holds the bulk of the cluster setup code

Let's defer thinking about this refactor until the future, when there's a real need and just see if we can relocate the neon service map code to **Neon.Kube** for now.

**Comment by jefflill** [comment]
@marcusbooyah: I'll take care of this.

**Comment by jefflill** [comment]
I went ahead and spent an extra 20 minutes doing the **_right thing_** and created and **Neon.Kube.Services** library and relocated these definitions there.  We'll put any common code for cluster services in here.

@marcusbooyah: I've added this to the official nuget publishing scripts as well as my personal `quick-neonkube-nuget-dev.ps1` script in neonCLOUD.  You'll want to add this to your _quick script_, if you have one.
